src/System.Management.Automation/engine/remoting/common/WireDataFormat/EncodeAndDecode.cs

```diff
@@ -2401,7 +2387,7 @@ internal static RemoteSessionCapability GetSessionCapability(object data)
                 //
                 // So just return the CurrentTimeZone.
 
-#if !CORECLR // TimeZone Not In CoreCLR
+#if !CORECLR // TimeZone deprecated, need to change to TimeZoneInfo
```

Comment:
Maybe add that this change affects serialization, remoting and backward compatibility. We can go back to that when we'll have tests for the areas.

Reply:
Good suggestion, will add